### DISCUSSION TITLE
[feature] show original path on disk

### DISCUSSION THREAD

**Discussion by mmomjian** [immich-app/immich#14353] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When showing the file path (the i icon on the file details), it would be cool to have a feature to substitute out upload/ with the user-provided original path on disk.

Example: UPLOAD_LOCATION=/mnt/immich

In Immich settings or env var, we specify "IMMICH_ROOT_PATH=/mnt/immich" (technically, this var would not be needed, because of the way that Immich currently handles env vars, UPLOAD_LOCATION is available to the container).

Implementation: conceptually, `sed "s;^upload;$UPLOAD_LOCATION;" <originalpath>`

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
I'm not sure how I feel about this. Making Immich aware of the world outside the container doesn't feel right, and in particular relying on an env var like `UPLOAD_LOCATION` that isn't known by the app and only exists in our specific compose setup doesn't seem great.

**Comment by mmomjian** [comment]
I think the use of UPLOAD_LOCATION is not strictly the best way, but whether Immich is aware of the world outside the container, the user is, so I still think there’s a benefit to this.

**Comment by bo0tzz** [comment]
Now that the default mount is /data, I'd say the displayed path is clear enough as is.

**Reply by mmomjian** [reply]
Yeah, I agree, especially since we hope to deprecate UPLOAD_LOCATION soonTM
